LICENSES/LicenseRef-Slint-Royalty-free-1.2.md

```diff
@@ -28,7 +30,7 @@ You may distribute the Software as part of an Application, modified or unmodifie
 
 ## 3. Limitations
 
-The license does not permit to distribute or make the Software publicly available alone and without integration into an Application. For this purpose you may use the Software under the GNU General Public License, version 3.
+The License does not permit to sell, rent, or lease the Software, or distribute or make the Software publicly available alone and without integration into an Application. For this purpose you may use the Software under the GNU General Public License, version 3.
```

Comment:
```suggestion
The License does not permit to sell, rent, or lease the Software. Additionally you may not distribute or make the Software publicly available alone and without integration into an Application. For this purpose you may use the Software under the GNU General Public License, version 3.
```

This is two different statements right? "available alone and without integration into an Application" refers only to the second one? Maybe better to split them.

It seems to prohibit you from selling the Application that integrates Slint.
Do you want to allow using this license for commercial purposes in general?
I don't think there is a clear way of prohibiting selling GUI toolkits that use Slint while allowing most of the typical applications, because that is a very subjective thing.

If you want to permit selling: I've looked at how the non-free licenses worded https://www.gnu.org/licenses/license-list.html and I think what you should do instead is to add termination clause and/or request user to receive permission to sell the software as part of the application. This allows for SixtyFPS to decide to terminate "inappropriate" uses of the license if they wish, post-factum and/or beforehand. Of course this seems like a rule that is too powerful, you should mention specific cases for when SixtyFPS may decide to terminate the program if you decide to add that.

Reply:
> Maybe better to split them.

Yes, that would be better. Thanks.

> Do you want to allow using this license for commercial purposes in general?

Yes, we want to allow building commercial / proprietary applications.

> I don't think there is a clear way of prohibiting selling GUI toolkits that use Slint ...

I think a GUI toolkit that uses Slint would not be considered as an application rather it would be classified as an SDK, right? And this license only permits creation of an application.

Reply:
> I think a GUI toolkit that uses Slint would not be considered as an application rather it would be classified as an SDK, right? And this license only permits creation of an application.

To make it clear, my points were that:
1. With how Application is defined here it would include libraries
2. Quite a lot of SDKs are just a regular GUI applications, for example game engines (editors), embedded tookits (speaking of vivado), some GUI toolkits (although not as popular there), care is needed to handle such cases
3. Term SDK is not defined here, and it is as clear to me as the difference between a "library" and the "framework" (not clear at all, is web browser a SDK?)

The points were addressed more or less in the new version.